src/Microsoft.ML.StandardLearners/Standard/SdcaMultiClass.cs

```diff
@@ -57,41 +57,36 @@ public sealed class Arguments : ArgumentsBase
         /// <param name="l2Const">The L2 regularization hyperparameter.</param>
         /// <param name="l1Threshold">The L1 regularization hyperparameter. Higher values will tend to lead to more sparse model.</param>
         /// <param name="maxIterations">The maximum number of passes to perform over the data.</param>
-        /// <param name="advancedSettings">A delegate to set more settings.
-        /// The settings here will override the ones provided in the direct method signature,
-        /// if both are present and have different values.
-        /// The columns names, however need to be provided directly, not through the <paramref name="advancedSettings"/>.</param>
         public SdcaMultiClassTrainer(IHostEnvironment env,
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
             string weights = null,
             ISupportSdcaClassificationLoss loss = null,
             float? l2Const = null,
             float? l1Threshold = null,
-            int? maxIterations = null,
-            Action<Arguments> advancedSettings = null)
-             : base(env, featureColumn, TrainerUtils.MakeU4ScalarColumn(labelColumn), TrainerUtils.MakeR4ScalarWeightColumn(weights), advancedSettings,
-                  l2Const, l1Threshold, maxIterations)
+            int? maxIterations = null)
+             : base(env, featureColumn, TrainerUtils.MakeU4ScalarColumn(labelColumn), TrainerUtils.MakeR4ScalarWeightColumn(weights),
+                   l2Const, l1Threshold, maxIterations)
         {
             Host.CheckNonEmpty(featureColumn, nameof(featureColumn));
             Host.CheckNonEmpty(labelColumn, nameof(labelColumn));
             _loss = loss ?? Args.LossFunction.CreateComponent(env);
             Loss = _loss;
         }
 
-        internal SdcaMultiClassTrainer(IHostEnvironment env, Arguments args,
+        internal SdcaMultiClassTrainer(IHostEnvironment env, Options options,
```

Comment:
>internal SdcaMultiClassTrainer [](start = 8, length = 30)

delete, maybe

Reply:
#2100

---
In reply to: [248770420](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248770420) [](ancestors = 248770420)